### ISSUE TITLE
Definition of Done

### ISSUE THREAD

**Issue by tawfeeka** [amosproj/amos2023ss02-open-search-meta-data-hub#21]
## User story
1. As a team
2. We need a Definition of Done that specifies the agreed-upon acceptance criteria for implementing a backlog item
3. This will ensure that all members have a shared understanding of the task and the acceptance criteria.

## Acceptance criteria
* This should be well documented in the planning document.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by motrell** [comment]
Sounds good!
